### ISSUE TITLE
Group of sprites where one acts as the collider and move together

### ISSUE THREAD

**Issue by itulau** [AustinEast/echo-flixel#2]
Hi, I've been trying out Echo and I like it so far, though there is one thing that I couldn't find in the demos to see if it's currently doable: I'm trying to create a player that has one central Body that acts as the collider that interacts with the walls, and within itself has a "weapon" collider that interacts only with enemies.  

I've tried using FlxSpriteGroup and called "add_body()" on it, and added the weapon sprite to the group and the problem I'm facing now is that the applied velocity to the "central" body doesn't get carried to the inner elements in the FlxSpriteGroup by default, so the weapon just stays in the same place.

Basically what I've need is a traditional hierarchy group, you have the parent to be the collider and the child just follows along. 

The "Bodies With Multiple Shapes" demo is kind of near what I'm trying to achieve, but instead the other shapes shouldn't collide but follow along the one that does collide, and ideally they should be sprites instead of bodies.

Do anyone know if this is doable with Echo/Echo-Flixel ?
